### DISCUSSION TITLE
Protocoll WH1080 (FSK)

### DISCUSSION THREAD

**Discussion by Micha70** [merbanan/rtl_433#2474] [General]
Hi, 
I'm searching protocoll data for weather station WH1080.

Last year I had found a very good documentation here: 
http://www.sevenwatt.com/main/wh1080-protocol-v2-fsk/

But unfortunately seems is offline .... :-(

Can anybody help me?

Regards
Michael

**Comment by merbanan** [comment]
Here you go:

https://web.archive.org/web/20220627175106/http://www.sevenwatt.com/main/wh1080-protocol-v2-fsk/
